docker-compose/monitoring/grafana/alerting/contact-points.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: 1
+contactPoints:
+  - name: liquidity-alerts-email
+    receivers:
+      - uid: email-receiver-01
+        type: email
+        settings:
+          addresses: "lps-alerts@localhost.com"
+          subject: "URGENT: LPS Liquidity Shortage"
+          body: |
```

Comment:
Is it possible to extract content from the log to be included in the body? This my be needed in some other alerts

Reply:
I'll research on this since its valuable to have some of the information logged by the LPS in the notification (for instance the quote hash).